log4j-over-slf4j/src/main/java/org/apache/log4j/Category.java

```diff
@@ -323,7 +323,8 @@ private int priorityToLevelInt(Priority p) {
         case Priority.FATAL_INT:
             return LocationAwareLogger.ERROR_INT;
         default:
-            throw new IllegalStateException("Unknown Priority " + p);
+            slf4jLogger.warn("Unknown Priority " + p);
+            return LocationAwareLogger.TRACE_INT;
```

Comment:
Original JIRA suggests DEBUG level. TRACE will be visible everywhere. So feel free to change the corrected logging level.